app/models/game.rb

```diff
@@ -43,4 +37,18 @@ def winning_team
     end
     team
   end
+
+  def maximum_score
+    MAX_POINTS_PER_GAME
```

Comment:
@shanebonham 👀 at this. I did this because seeing constants like `Game::MAX_POINTS_PER_GAME` littered all over the app always feels like a code smell to me, like I don't think other objects should care how Game implemented max points, just that a Game can tell you what it's max score is. But a constant used like this inside the class that defined it does add some clarity.